### DISCUSSION TITLE
Port to python.

### DISCUSSION THREAD

**Discussion by dikeshmoody** [rstudio/pointblank#413] [Ideas]
Hi,

Do we have any plans to port it to python?

**Comment by rich-iannone** [comment]
There is a need and a rough plan for this. A few things are currently missing, like easy table generation for output reports (but that can be added later). Basically, it would be really good to start this project and build out the basic (validation and test) checks as a first step. I'll do this in 1-2 days.

**Reply by dikeshmoody** [reply]
Sounds good. Let us collaborate on it.

**Reply by rich-iannone** [reply]
Absolutely. I'm going to set everything up in a private repo for now and make it public once the basic setup is in place. More on this soon.

**Reply by lenapheno** [reply]
any updates? :)

**Comment by pachadotdev** [comment]
@rich-iannone hi, do you have any plans to take on this? I can start with a sketch

**Reply by rich-iannone** [reply]
I had hopes awhile back, but no real time to start this... I'd love it if you could get it off the ground! I have a repo for this, will invite you in a bit.

**Reply by lenapheno** [reply]
i will be happy to get involved and colloborate on this too

**Comment by m-freitag** [comment]
A python port would be neat, and possibly leaner than R. Also, I think there is actually a bit of a lack of a great OSS data validation framework in python. Great Expectations has a pretty clunky api, Soda is only good with the paid version.

**Comment by rich-iannone** [comment]
Python port is now available! https://github.com/posit-dev/pointblank

@dikeshmoody @m-freitag @lenapheno
